webpack/ForemanTasks/Components/TasksTable/TasksTableConstants.js

```diff
@@ -18,4 +18,4 @@ export const CANCEL_SELECTED_CONFIRM_MODAL_ID = 'cancelSelectedConfirmModal';
 export const RESUME_SELECTED_CONFIRM_MODAL_ID = 'resumeSelectedConfirmModal';
 export const UPDATE_CLICKED = 'UPDATE_CLICKED';
 
-export const TASKS_SEARCH_PROPS = getControllerSearchProps('tasks');
+export const TASKS_SEARCH_PROPS = getControllerSearchProps('foreman_tasks/tasks');
```

Comment:
This should be:
`export const TASKS_SEARCH_PROPS = {
  ...getControllerSearchProps('tasks'),
  controller: 'foreman_tasks/tasks',
};`
as the controller for the autocomplete is 'tasks' and for the search is 'foreman_tasks/tasks'